pkg/services/libraryelements/libraryelements_test.go

```diff
@@ -210,80 +139,6 @@ func TestIntegration_GetLibraryPanelConnections(t *testing.T) {
 			}
 		})
 
-	scenarioWithPanel(t, "When a user tries to get connections of library panel, dashboards in inaccessible folders should not be returned",
```

Comment:
moved to pkg/tests/apis/dashboard/integration/library_panels_api_validation_test.go

Reply:
Moved to https://github.com/grafana/grafana/pull/108737